src/Microsoft.TemplateEngine.Edge/Settings/TemplateCache.cs

```diff
@@ -57,10 +69,23 @@ public TemplateCache(IEngineEnvironmentSettings environmentSettings, JObject par
         [JsonProperty]
         public List<TemplateInfo> TemplateInfo { get; set; }
 
+        [JsonProperty]
+        public string CacheVersion { get; set; }
+
+        public static bool CheckIfCacheVersionIsCurrent(string cacheVersion)
```

Comment:
If this were an instance method, it could just be a property & compare to the static/const field indicating the current version number caches would be written with